source/include/vx_reference.h

```diff
@@ -55,6 +55,16 @@ typedef vx_status (*tivx_reference_callback_f)(vx_reference ref);
  */
 typedef vx_status (* VX_API_CALL tivx_reference_release_callback_f)(vx_reference *ref);
 
+/*! \brief Callback type used to validate or implement kernels for operations
+ * involving generic references, such as Copy, Select, Swap or Pass
+ * \param [in] kernel_enum        A <tt>\ vx_kernel_e </tt> which give the name of vision kernel
+ * \param [in] validate_only      A <tt>\ref vx_bool</tt> used to validate if the kernel operation is possible or not
+ * \param [in] input              The pointer to a \ref vx_references for the input and outputs used for the kernel operations
+ * \param [in] output             The pointer to a \ref vx_references for the output used for the kernel operations
```

Comment:
@stephenramm With the API change recently, is this still "vx_references", or a singular "vx_reference" in the documentation of this parameter?

Reply:
documentation modified, this is a single ref for the input and single ref for the output